src/Admin/Menu/MenuNodeAdmin.php

```diff
@@ -69,14 +70,24 @@ protected function configureFormFields(FormMapper $formMapper)
                                 'content' => 'content',
                             ),
                             'map' => array(
-                                'route' => array('link'),
+                                'route' => array('link','routeParameters'),
                                 'uri' => array('link'),
                                 'content' => array('content', TreeSelectType::class),
                             ),
                             'placeholder' => 'auto',
                             'required' => false,
                         ))
                         ->add('link', TextType::class, array('required' => false, 'mapped' => false))
+                        ->add('routeParameters', KeyValueType::class, array(
```

Comment:
i would only make the route parameters optional. the rest works without the key-value bundle